src/Pos/Aeson/Types.hs

```diff
@@ -42,6 +43,9 @@ instance ToJSON TxOut where
         "coin"    .= coinToInteger txOutValue,
         "address" .= sformat build txOutAddress ]
 
+deriving instance ToJSON SlotAmount
+deriving instance ToJSON BlockAmount
+
 -- NOTE: some of these types are used on frontend (PureScript).
```

Comment:
You might have broken something in communication with frontend, but I am not an expert in this area.